pkg/services/serviceaccounts/database/token_store.go

```diff
@@ -0,0 +1,63 @@
+package database
+
+import (
+	"context"
+	"time"
+
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+)
+
+func (s *ServiceAccountsStoreImpl) AddServiceAccountToken(ctx context.Context, cmd *models.AddApiKeyCommand) error {
+	return s.sqlStore.WithTransactionalDbSession(ctx, func(sess *sqlstore.DBSession) error {
+		key := models.ApiKey{OrgId: cmd.OrgId, Name: cmd.Name}
```

Comment:
thought: would it also be worth creating separate models for SATs? Otherwise the code is slightly confusing. But maybe it's not a good idea if we're still using one table for both SATs and API keys.

Reply:
I'd keep it as it is while we still use one table (model == table), otherwise there would also be no indication they share the same backend, which might be dangerous